File: src/Web/WebAngular/src/app/domain/view-models/connector.view-model.ts
import { ConnectorFunctionViewModel } from "./connector-function.view-model";

export interface ConnectorViewModel {
	id: string;
	name: string;
	description: string;
	connectorFunctions: ConnectorFunctionViewModel[];
	createdBy: string;
	creationDate: string;
	updatedBy: string;
	lastUpdate: string;
}

File: src/Core/Houston.Application/ViewModel/ConnectorViewModels/ConnectorViewModel.cs
namespace Houston.Application.ViewModel.ConnectorViewModels {
	public class ConnectorViewModel {
		public Guid Id {  get; set; }

		public string Name { get; set; } = null!;

		public string FriendlyName { get; set; } = null!;

		public string? Description { get; set; }

		public string CreatedBy { get; set; } = null!;

		public DateTime CreationDate { get; set; }

		public string UpdatedBy { get; set; } = null!;

		public DateTime LastUpdate {  get; set; }
	}
}
